DataFormats/Scouting/interface/Run3ScoutingVertex.h

```diff
@@ -11,9 +11,12 @@ class Run3ScoutingVertex {
   Run3ScoutingVertex(float x,
                      float y,
                      float z,
-                     float zError,
                      float xError,
                      float yError,
+                     float zError,
+                     float xyCov,
+                     float xzCov,
+                     float yzCov,
```

Comment:
I have the same comment as https://github.com/cms-sw/cmssw/pull/43744#discussion_r1458891652. Any reason to not pass it to the constructor as a more compact object?

Reply:
This was done in line with all other scouting classes.
I see the point of the comment. IMHO, it may be worth revisiting all scouting classes at once, using homogeneous schemes for compact constructors. This may be an item for scouting conveners to discuss further, though, independently of this PR.